tools/CoherenceBuild/CoherenceBuild.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
```

Comment:
I doubt 1.x is required, but this was the minimal change required to keep it working after https://github.com/aspnet/BuildTools/issues/347